tests/CMakeLists.txt

```diff
@@ -537,6 +632,19 @@ foreach(pol IN LISTS DSLASH_POLICIES)
                      --dim 2 4 6 8
                      --Lsdim 4
                      --gtest_output=xml:dslash_mobius_test_pol${pol2}.xml)
+    
+    add_test(NAME benchmark_dslash_${DIRAC_NAME}-policy${pol2}
+    COMMAND ${QUDA_CTEST_LAUNCH} $<TARGET_FILE:dslash_ctest> ${MPIEXEC_POSTFLAGS}
+            --dslash-type ${DIRAC_NAME}
+            --test 0
+            --dim 20 20 20 20
```

Comment:
Add a specific `--Lsdim` --- maybe 16? @hummingtree

Reply:
(applicable to other dwf types too)

Reply:
I used 12 here for now.

Reply:
`12` should be good here.